src/screens/ToDoList/ToDoListItem/styl.scss

```diff
@@ -0,0 +1,63 @@
+@mixin row-form {
+  display: flex;
+  flex-direction: row;
+}
+
+@mixin button-check-delete {
+  border-radius: 50%;
+  outline: 0px;
+  cursor: pointer;
+  margin-left: 5px;
+  height: 17px;
+  width: 17px;
+}
+
+.row-item {
+  @include row-form;
+  margin-left: 50px;
+  align-items: center;
+}
+
+.row-item-completed {
+  @include row-form;
+  position: relative;
+  outline: 0;
+  cursor: pointer;
+  opacity: .5;
+  text-decoration: line-through; 
+  margin-left: 50px; 
+  align-items: center;
+}
+
+.row{
+  @include row-form;
+  align-items: center;
+}
+
+.delete-button {
+  @include button-check-delete;
+  background-color: white;
+  border: 1px solid red;
+  color:black;
+  margin-top: 5px;
+  padding: 0px 0px 2px 2px ;
+  margin-left: 14px;
+}
+
+.delete-button:hover {
+  background-color: red;
+}
+
+.check-button {
+  @include button-check-delete;
+  border: 1px solid green;
+  margin-bottom: 15px;
+  background-color: green;
```

Comment:
You should create separate file where you can keep all colors and later just import them anywhere.